src/StringTools/StringTools.csproj

```diff
@@ -14,6 +13,11 @@
 
     <AssemblyName>Microsoft.NET.StringTools</AssemblyName>
     <PackageDescription>This package contains the $(AssemblyName) assembly which implements common string-related functionality such as weak interning.</PackageDescription>
+    
+    <IncludeBuildOutput Condition="'$(TargetFramework)' == 'net35'">false</IncludeBuildOutput>
+    <!-- Don't publish the reference assembly if the build output isn't included. -->
+    <TargetsForTfmSpecificBuildOutput Condition="'$(IncludeBuildOutput)' != 'true'" />
```

Comment:
Isn't `IncludeBuildOutput` empty here by default? With this change the package has only netstandard2.0 ref assembly.

![image](https://user-images.githubusercontent.com/12206368/204383370-3561c1af-65ba-45d3-baad-f322a6f5cff5.png)